### DISCUSSION TITLE
Using Topology views with API

### DISCUSSION THREAD

**Discussion by betadrome** [mattieserver/netbox-topology-views#569] [Q&A]
Hey, is it possible to use API methods GET, POST, PUT, PATCH, DELETE for the models `Cordinate Groups` and `Device Cordinates`? Unfortunately I couldn't find anything in the Swagger GUI and here on Github

**Comment by dreng** [comment]
This is not possible yet. Feel free to create a Feature Request.

**Comment by dreng** [comment]
See #570
